src/Abp/WebHooks/WebHookWorkItem.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Net;
+using Abp.Domain.Entities;
+using Abp.Domain.Entities.Auditing;
+
+namespace Abp.WebHooks
+{
+    /// <summary>
+    /// Table for store webhook work items. Each item stores web hook send attempt of <see cref="WebHookInfo"/> to subscribed users
+    /// </summary>
+    [Table("AbpWebHookWorkItems")]
+    public class WebHookWorkItem : Entity<Guid>, IMayHaveTenant, IHasCreationTime, IHasModificationTime, ISoftDelete
```

Comment:
Probably it is better to name it `WebhookSendAttempt` directly. Similar to `UserLoginAttempt`

Reply:
Yes, it looks better 👍  
Choosing a new name is one of the most difficult things in developing 😄